pkgs/servers/foundationdb/default.nix

```diff
@@ -76,13 +76,14 @@ in with builtins; {
   # ------------------------------------------------------
 
   foundationdb61 = cmakeBuild {
-    version = "6.1.12";
+    version = "6.1.13";
     branch  = "release-6.1";
-    sha256  = "1yh5hx6rim41m0dwhnb2pcwz67wlnk0zwvyw845d36b29gwy58ab";
+    sha256  = "10vd694dcnh2pp91mri1m80kfbwjanhiy50c53c5ncqfa6pwvk00";
 
     patches = [
       ./patches/clang-libcxx.patch
       ./patches/suppress-clang-warnings.patch
+      ./patches/stdexcept-6.1.patch
```

Comment:
Is there an upstream patch we can fetch instead?

Reply:
No, as 6.1.13 seems the last release of the FoundationDB 6.1 series, published on 2020-05-04. I will ask upstream anyway. Thanks!

Reply:
Posted upstream: https://forums.foundationdb.org/t/eol-policy-of-foundationdb/2581